docs/cdoc/index.h

```diff
@@ -13,3 +13,11 @@
 /**
  * @defgroup QkCircuit QkCircuit
  */
+
+/**
+ * @defgroup QkCircuit QkQuantumRegister
+ */
+
+/**
+ * @defgroup QkCircuit QkClassicalRegister
+ */
```

Comment:
I think for this you'd want to keep the same names in the group definition IIRC
```suggestion

/**
 * @defgroup QkQuantumRegister QkQuantumRegister
 */

/**
 * @defgroup QkClassicalRegister QkClassicalRegister
 */
```